filebeat/_meta/common.full.p2.yml

```diff
@@ -215,6 +215,14 @@ filebeat.prospectors:
 # Configuration to use stdin input
 #- input_type: stdin
 
+#----------------------------- Redis prospector -------------------------------
+#- input_type: redis
```

Comment:
I suggest somehow making it more clear that this is strictly about slow logs. People might assume that it's a general Redis input.